src/index.ts

```diff
@@ -1,100 +1,79 @@
 import { Elysia, t } from "elysia"
-
-let data = [{
-  id: 1,
-  title: 'First Item',
-  description: "Description for the first item",
-  isComplete: false
-},
-{
-  id: 2,
-  title: 'Second Item',
-  description: "Description for the second item",
-  isComplete: false
-},
-{
-  id: 3,
-  title: 'Third Item',
-  description: "Description for the third item",
-  isComplete: false
-}]
+import {cors} from '@elysiajs/cors'
+import { databasePlugin } from "./decorators/db"
 
 const app = new Elysia()
-.state('version',1)
-.decorate('getTodoById', (id: number)=> data.find(item => item.id === id))
-.decorate('addNewTodoItem', (title: string, description: string) => {
-  const newObject: { id: number, title: string, description: string, isComplete: boolean } = {
-    id: 0,
-    title,
-    description,
-    isComplete: false
-  }
-
-  const lastItem = data[data.length - 1]
-  const newId = lastItem ? lastItem.id + 1 : 1
+.use(cors())
+.use(databasePlugin)
 
-  newObject.id = newId
-
-  // Add the new object to the data array
-  data.push(newObject)
+app.state('version',1)
+.get("/all", ({ dbAction, database })=> {
```

Comment:
```suggestion
.get("/all", ({ dbAction, database }) => {
```
I see some indenting things through the code, maybe interesting to have `prettier` to fix this.  https://prettier.io/